c/README.md

```diff
@@ -0,0 +1,39 @@
+# Echo Bot - C
+
+## Requirements
+
+- This readme describes the process for compiling on **linux**, if you are on another platform your steps might vary. If you figgured out how to do it on your platform feel free to sumbit a pr to add your method to this readme.
+- You need `libdeltachat`,`git`, `cmake`, `gcc` and `pkg-config` also `rustup` if you compile `libdeltachat` on your own
+
+### Installing libdeltachat from source
+
+```sh
+git clone https://github.com/deltachat/deltachat-core-rust
+cd deltachat-core-rust
+cmake -B build . && cmake --build build && sudo cmake --install build
+```
+
+> Info: There is no uninstall yet, use `sudo rm` to remove the files cmake created.
```

Comment:
Replace this with "To uninstall, run `sudo xargs -d\\n rm -f <build/install_manifest.txt`".